# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContextCGL.h
# Commit: 204b899f436d
# Full Hash: 204b899f436d99f0eae759af3ccfcbaabfbbb428
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 09:34:54
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 3f5baab34b1f -r 204b899f436d gfx/gl/GLContextCGL.h
--- a/gfx/gl/GLContextCGL.h	Thu Jun 11 06:36:55 2020 +0000
+++ b/gfx/gl/GLContextCGL.h	Thu Jun 11 06:37:35 2020 +0000
@@ -35,8 +35,7 @@
 
  public:
   MOZ_DECLARE_REFCOUNTED_VIRTUAL_TYPENAME(GLContextCGL, override)
-  GLContextCGL(CreateContextFlags flags, const SurfaceCaps& caps,
-               NSOpenGLContext* context, bool isOffscreen);
+  GLContextCGL(const GLContextDesc&, NSOpenGLContext* context);
 
   ~GLContextCGL();
 